core-bundle/src/Resources/contao/controllers/BackendPassword.php

```diff
@@ -55,8 +55,14 @@ public function run()
 		{
 			$pw = $request->request->get('password');
 
+			// Make sure the password gets evaluated against custom regex if they exist
+			// First rule, as these requirements are usually stronger than the default ones
+			if ($message = self::_evalRegexp($pw))
+			{
+				Message::addError($message);
+			}
 			// Password too short
-			if (mb_strlen($pw) < Config::get('minPasswordLength'))
+			elseif (mb_strlen($pw) < Config::get('minPasswordLength'))
```

Comment:
to be discussed if this and other occurrences should be `else` conditions, as the user might be interested in a list of validation issues to fix them all together with a new password and not one by one...